ansible/test/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Bring up docker containers
+  hosts: localhost
+  gather_facts: false
+  vars:
+    inventory:
+      - name: smtp-outbound
+        image: "geerlingguy/docker-centos7-ansible"
```

Comment:
any reason for this instead of the basic `centos:centos7` container?

Reply:
Replaced with: image: "centos:centos7"

Reply:
I spotted that Matt had used the docker container in his and thats likely where you got it from.
Happy to use it as its what the base was tested on but either should work for our use case.

Reply:
Reverted back to image: "geerlingguy/docker-centos7-ansible" as I was getting the following error message when starting services:

Failed to get D-Bus connection: Operation not permitted

I added the code from the latest Centos 7 base AMI for the docker test dir:

image: "geerlingguy/docker-centos7-ansible"
        # The network setting allows for idempotent testing.
        # without it the role creates a brand new container, with it the container is re-used.
        # network_mode: bridge
        provision_docker_volumes: 
          - /lib/modules:/lib/modules